route/note-route.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
```

Comment:
This is good, but hasn't been refactored to utilize your storage module.  Please see my comments below for more information.